src/messages.ts

```diff
@@ -456,32 +456,60 @@ export const bottomMenuMessages = defineMessages('ui.src.bottomMenu', {
   },
 });
 
+export const purposeMessages = defineMessages('purpose.trackingType', {
+  'essential.title': {
```

Comment:
```suggestion
  'Essential.title': {
```
these tracking types should all be pascal case, right? we switched it in the monorepo

Reply:
oh right, I need to fix this. weirdly it still worked

Reply:
yeah because the ones being displayed are overwritten successfully. if you remember when we were testing we had like `Functional: undefined`, p sure that was because of this

Reply:
oh, yeah I think you're right!